cico_run_tests.sh

```diff
@@ -1,12 +1,11 @@
 
 #!/bin/bash
 
-. cico_setup.sh
+. ./cico_setup.sh
```

Comment:
Verified?

Reply:
Does this statement import cico_setup.sh to satisfy push image line below?
Why do you want to run it from here ?

Reply:
With  >" . cico_setup.sh" ,
I'm getting errors:

./cico_run_tests.sh: line 4: .: cico_setup.sh: file not found